Automattic/wp-calypso
# Jetpack Connect: Remove problematic "blacklist"/"whitelist" language #43395

#### Changes proposed in this Pull Request

* Rename `SITE_BLACKLISTED` to `SITE_BLOCKED`
* Rename `isSiteBlacklistedError` to `isSiteBlockedError`
* Rename `MOBILE_APP_REDIRECT_URL_WHITELIST` to `ALLOWED_MOBILE_APP_REDIRECT_URL_LIST`

#### Testing instructions

* Ensure blocked sites still behave as expected (are not allowed to connect to Jetpack, etc).

Addresses part of https://github.com/Automattic/wp-calypso/issues/43395
